src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodec.java

```diff
@@ -345,10 +350,15 @@ static ByteBuffer byteBufferFromEJSON(DataType targetCQLType, EJSONWrapper wrapp
         if (value.isBinary()) {
```

Comment:
Can we add a test case to pass the value as binary?

Reply:
Hmmh. Let me think about that... it is not possible for JSON to produce `BinaryNode` (it is only possible via custom nodes or binary formats -- like CBOR or Smile).

But maybe I can hand-code it. Let me see about that.

Reply:
Unfortunately not quite as easy to push through plumbing. Will need to defer this (but leaving "binary" handling in there since it was not added in this pr.